app/models/concerns/tiny_mce_images.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module TinyMceImages
+  extend ActiveSupport::Concern
+
+  included do
+    has_many :tiny_mce_assets,
+             as: :object,
+             class_name: :TinyMceAsset,
+             dependent: :destroy
+
+    def prepare_for_report(field)
+      images = tiny_mce_assets
+      description = self[field]
+      images.each do |image|
+        tmp_f = Tempfile.open(image.image_file_name, Rails.root.join('tmp'))
+        begin
+          image.image.copy_to_local_file(:large, tmp_f.path)
```

Comment:
A bit misleading naming, may be it is better to use tiny_mce_asset instead of image?